g11n-ws/modules/md-restful-synch/build.gradle

```diff
@@ -46,9 +46,8 @@ dependencies {
     compile project(":md-restful-api")
     compile project(":md-service-i18n-l3")
     compile("org.apache.httpcomponents:httpclient:4.5.6")
-    compile("com.alibaba:fastjson:1.2.51")
-    compile("io.springfox:springfox-swagger-ui:2.5.0")
-    compile("io.springfox:springfox-swagger2:2.5.0")
+    compile("io.springfox:springfox-swagger-ui:2.9.2")
+    compile("io.springfox:springfox-swagger2:2.9.2")
```

Comment:
Why change swagger-ui? Please create a new PR for it.

Reply:
I will revert it